File: src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/SubmissionSentNotification.java
package fi.aalto.cs.apluscourses.intellij.notifications;

import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getText;

import com.intellij.notification.Notification;
import com.intellij.notification.NotificationType;
import fi.aalto.cs.apluscourses.services.PluginSettings;

public class SubmissionSentNotification extends Notification {

  /**
   * Construct a notification which informs the user that an exercise was submitted successfully.
   */
  public SubmissionSentNotification() {
    super(
        PluginSettings.A_PLUS,
        getText("notification.SuccessfulSubmissionNotification.title"),
        getText("notification.SuccessfulSubmissionNotification.content"),
        NotificationType.INFORMATION
    );
  }

}

File: src/main/kotlin/fi/aalto/cs/apluscourses/notifications/SubmissionSentNotification.kt
package fi.aalto.cs.apluscourses.notifications

import com.intellij.notification.Notification
import com.intellij.notification.NotificationType
import fi.aalto.cs.apluscourses.MyBundle
import fi.aalto.cs.apluscourses.services.PluginSettings

class SubmissionSentNotification
/**
 * Construct a notification which informs the user that an exercise was submitted successfully.
 */
    : Notification(
    PluginSettings.A_PLUS,
    MyBundle.message("notification.SuccessfulSubmissionNotification.title"),
    MyBundle.message("notification.SuccessfulSubmissionNotification.content"),
    NotificationType.INFORMATION
)
